src/renderers/dom/test/ReactTestUtilsEntry.js

```diff
@@ -437,16 +437,16 @@ var ReactTestUtils = {
 /**
  * Exports:
  *
- * - `ReactTestUtils.Simulate.click(Element)`
- * - `ReactTestUtils.Simulate.mouseMove(Element)`
- * - `ReactTestUtils.Simulate.change(Element)`
+ * - `ReactTestUtils.Simulate.click(domNode)`
```

Comment:
Minor nit but can we keep `Element` here? It refers to DOM `Element` type (which is what DOM node is), not to argument name.